### ISSUE TITLE
Async select doesn't get repositioned after loading is done

### ISSUE THREAD

**Issue by vincaslt** [JedWatson/react-select#5042]
This seems like a bug, I saw some discussions about using popper, but seems like custom implementation was chosen. Popper would accept handleResize, which I think fixes this. Does your implementation have a property to observe resize and update position accordingly?

Issue: position is calculated during loading state, after data is loaded, it's not recalculated resulting on off-screen content.
recording: https://www.loom.com/share/ea00e7c4364649e4ace2ec87f6a51a95

**Comment by Rall3n** [comment]
Hello @vincaslt,

would you please be so kind and include a minimal reproducible example using [CodeSandbox](https://codesandbox.io/s/react-select-v5-sandbox-y5jtm) **as per the issue template**?

**Comment by vincaslt** [comment]
Absolutely, here you go: https://codesandbox.io/s/react-select-v5-sandbox-forked-6vtdj?file=/example.js

**Comment by vincaslt** [comment]
Any news on this?

**Comment by labkey-nicka** [comment]
#5256 provided a new implementation for how/when resizing is calculated for the menu. @Methuselah96 do you have any thoughts on how `runAutoUpdate()` could be invoked again after an async load?

**Comment by labkey-nicka** [comment]
A newer issue, specifically #5733, is more the point of what would need to be updated with the current version of react-select (v5.7.4).
